src/middleware/authenticated.ts

```diff
@@ -0,0 +1,44 @@
+import jwt from 'jsonwebtoken';
+import { JWT_SECRET } from '../siteConstants';
+import { DeveloperDocument, Express } from '../types';
+import { createDbConnection, handleCloseConnection } from '../services/database';
+import { developerSchema } from '../models/Developer';
+
+export const isAuthenticated: Express.MiddleWare = async (req, res, next) => {
+  let token: string | undefined;
+  // check if token is set
```

Comment:
nit: please start comments with uppercase
```suggestion
  // Check if token is set
```